src/Game.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+function Game(){
+  this.MAX_FRAMES = 10;
+  this.frames = [];
+  this.newFrame = new Frame(1);
+  this.totalscore = 0;
+}
+
+Game.prototype._canBowl = function() {
```

Comment:
This method throws an exception if there's a problem, but also returns true if there's no problem.  This feels like two responsibilities.  Can we use the lack of an exception as a sign that all is well, instead?